.ci/unit-tests/inspec.sh

```diff
@@ -49,15 +33,17 @@ erb -r './var' inspec.json.erb > inspec.json
 export GOOGLE_APPLICATION_CREDENTIALS=${PWD}/inspec.json
 
 bundle install
-# TODO change this to use a github repo
-gsutil cp -r gs://magic-modules-inspec-bucket/inspec-cassettes .
+git clone https://github.com/modular-magician/inspec-gcp-cassettes
```

Comment:
This is a public repo right now.

That means that (hopefully) everything in this repo is safe for public consumption. If that's true, what would the downsides be to placing the cassettes directly inside the spec/ folder?